requirements.txt

```diff
@@ -1,3 +1,4 @@
 tensorflow==2.8.0
 ktrain==0.30.0
-cached-path==1.1.1
\ No newline at end of file
+cached-path==1.1.1
+streamlit==1.8.1
```

Comment:
```suggestion
```

I wouldn't add streamlit here be honest, it's a whole bunch of dependencies that the user probably doesn't need. I would just add a new section to the readme explaining to the user how to run the app. Also, would rename `Accessory_App.py` to `app.py`. Both to follow lowercase convetions and because this is what this sort of thing is usually named.

Reply:
I'm fine with this but wouldn't this prevent us from deploying the app?

Reply:
I don't know if the streamlit cloud environment comes with streamlit installed (I would think so?). If not we can organize it like this:

pathway-abstract-classifier
---- requirements.txt
---- app
-------- app.py
-------- requirements.txt <-- this has everything in the main requirements + streamlit

Reply:
Ah, I wasn't able to do it because I'm not `PathwayCommons`. @jvwong or @maxkfranz might have to do this.

Reply:
__Edit__: Yeah so this works, I just tried it! This can be merged.

Reply:
@JohnGiorgi okay great! So what URL or instructions should I be adding to the README?

Reply:
Want to merge this and then I can add those URLs in another PR? I might do a small re-org of the repo